<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LogViewDlg</class>
 <widget class="QDialog" name="LogViewDlg">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>800</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>view log</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="2" column="3">
    <widget class="QPushButton" name="mCloseBtn">
     <property name="text">
      <string>Close</string>
     </property>
    </widget>
   </item>
   <item row="2" column="2">
    <widget class="QPushButton" name="mClearBtn">
     <property name="text">
      <string>Clear</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QPushButton" name="mSaveBtn">
     <property name="text">
      <string>Save</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QComboBox" name="mLogLevelCombo"/>
   </item>
   <item row="1" column="0" colspan="4">
    <widget class="QTextBrowser" name="mLogText"/>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>mLogText</tabstop>
  <tabstop>mLogLevelCombo</tabstop>
  <tabstop>mSaveBtn</tabstop>
  <tabstop>mClearBtn</tabstop>
  <tabstop>mCloseBtn</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
